dispatcher.rb

```diff
@@ -30,10 +34,19 @@ def call(message)
 
   attr_reader :bot
 
-  def dispatch_callback(message)
-    if message.data == 'like'
+  def dispatch_callback(callback)
+    begin
+      command = JSON.parse(callback.data)['command']
+    rescue JSON::ParserError
+      return nil
+    end
+
+    if command == 'like'
       # redis.publish('liker_bot', message.from.username)
-      bot.api.answer_callback_query(callback_query_id: message.id)
+      # Dispatchers::Like.new(...)
+      bot.api.answer_callback_query(callback_query_id: callback.id)
+    elsif command == 'more'
```

Comment:
I suggest using "job_more" or something with `job` prefix here as it is global namespace of commands